### ISSUE TITLE
Tests break when orchestra/database 4.0 is used

### ISSUE THREAD

**Issue by riekelt** [404labfr/laravel-impersonate#93]
In `tests/TestCase.php` migrations are loaded with:

```php
$this->loadMigrationsFrom([
            '--database' => 'testbench',
            '--realpath' => realpath(__DIR__ . '/../migrations'),
]);
```

In `orchestra/database:4.0` which is referenced in composer.json the realpath parameter is throwing an exception

https://github.com/orchestral/testbench-core/commit/4bcc026837cddcf0fc1c7ba8cc5d621e43d55199#diff-c93be3071990c1e1026e683b30c90aa4

**Comment by MarceauKa** [comment]
Hi and thank you. Can you make a PR?

**Comment by MarceauKa** [comment]
Fixed in [1.7.2](https://github.com/404labfr/laravel-impersonate/releases/tag/1.7.2)
